src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -68,6 +106,8 @@ public IConsole Console
             }
         }
 
+        public Action OnFlush { get; set; }
```

Comment:
Could extend `ConsoleLogger` in tests and add it there instead of in the lib? Open to other suggestions.